recipe/meta.yaml

```
@@ -21,9 +21,9 @@ build:
   skip: True
   {% endif %}
   script:
-    - cp {{ os.environ["FEEDSTOCK_ROOT"] }}/LICENSE.txt ${RECIPE_DIR}/LICENSE.txt            # [linux]
-    - cp ${RECIPE_DIR}/../LICENSE.txt ${RECIPE_DIR}/LICENSE.txt                              # [osx]
-    - COPY "%RECIPE_DIR%\\..\\LICENSE.txt" "%RECIPE_DIR%\\LICENSE.txt"                       # [win]
+    - cp {{ os.environ.get("FEEDSTOCK_ROOT", "failme") }}/LICENSE.txt ${RECIPE_DIR}/LICENSE.txt  # [linux]
```

We might consider adding a guard here. Also we could flesh out the error message a bit

```suggestion
    {% if "FEEDSTOCK_ROOT" in os.environ %}
    - cp {{ os.environ["FEEDSTOCK_ROOT"] }}/LICENSE.txt ${RECIPE_DIR}/LICENSE.txt  # [linux]
    {% else %}
    - echo '${FEEDSTOCK_ROOT} is undefined. Cannot copy license file'
    - exit 1
    {% endif %}
```